articlequality/feature_lists/wikidatawiki.py

```diff
@@ -38,6 +38,7 @@ class items:
 def _process_references(entity):
     return [reference
             for pid, statements in entity.properties.items()
+            if pid in property_datatypes.NONEXTERNAL_IDENTIFIERS
```

Comment:
Non-external == Internal?

Reply:
The total number of identifiers is 8000+ and the amount of external identifiers is 5000+. If we only check for non-external ids we are comparing against the ~half of properties. 

In this case we don't want to count the references for external identifiers.

Reply:
Right.  It's just that the term "nonexternal" is strange.  Are all non-external identifiers "internal" and if so that might be a better term.

Reply:
It's not "non-external" identifier (i.e. "internal idetnifier", we don't' have such concept in wikidata's data model), it's non-"external identifier" (anything else, like numbers, dates, strings, items, etc.)

Reply:
Aha!  Makes sense.